.github/workflows/build.yml

```
@@ -14,16 +14,12 @@ jobs:
         with:
           distribution: temurin
           java-version: 17
-      - name: Build
+      - name: Build and Test
         uses: gradle/gradle-build-action@v2
         with:
-          arguments: build
+          arguments: build conformanceTests
```

```suggestion
          arguments: conformanceTests
```

Given your point about the dependency that's declared, would it make sense to request just `conformanceTests`? If nothing else, it could be nice to know if that request _stops_ working, especially if it's what we'd tell our (legions of) users to run.

---

Well, `build` depends on more than just `conformanceTests`, so your suggestion would not run all the verification tasks.

It used to be that the conformance tests, like the samples, were separate from the normal Gradle build lifecycle. But now they're not. `conformanceTests` is part of `check`, which is the lifecycle task for all verification, and which `build` depends on.

---

Oh, thanks. For starters, I did have it backward: While your message had said that `build` depends on `conformanceTests`, I read it the other way around (presumably from being used to the Bazel terminology).

I guess that my question _should_ have been whether we might want to pass only `build`?

---

I considered that. But I thought this would make it a little more explicit. I checked, and this doesn't run `conformanceTests` twice.

---

:+1: